common/src/main/java/com/discordsrv/common/config/main/MainConfig.java

```diff
@@ -80,6 +80,9 @@ public BaseChannelConfig createDefaultBaseChannel() {
     @Comment("Configuration for the %discord_invite% placeholder. The below options will be attempted in the order they are in")
     public DiscordInviteConfig invite = new DiscordInviteConfig();
 
+    @Comment("Configuration for the avatar URL to provide to webhook messages and embeds")
```

Comment:
This should say that it is for the `player_avatar_url` placeholder instead of 'just' webhook messages and embeds, because that placeholder can be used anywhere (where there is a player in context), even through PlaceholderAPI if somebody really wanted to